ignite/handlers/param_scheduler.py

```diff
@@ -788,20 +788,43 @@ class LRScheduler(ParamScheduler):
         save_history: whether to log the parameter values to
             `engine.state.param_history`, (default=False).
 
-    .. code-block:: python
+    Examples:
+
+        .. testsetup::
+
+            default_trainer = get_default_trainer()
+
+        .. testcode::
 
-        from ignite.handlers.param_scheduler import LRScheduler
-        from torch.optim.lr_scheduler import StepLR
+            from torch.optim.lr_scheduler import StepLR
+
+            torch_lr_scheduler = StepLR(default_optimizer, step_size=3, gamma=0.1)
+
+            scheduler = LRScheduler(torch_lr_scheduler)
+
+            # In this example, we assume to have installed PyTorch>=1.1.0
+            # (with new `torch.optim.lr_scheduler` behaviour) and
+            # we attach scheduler to Events.ITERATION_COMPLETED
+            # instead of Events.ITERATION_STARTED to make sure to use
+            # the first lr value from the optimizer, otherwise it is will be skipped:
+            default_trainer.add_event_handler(Events.ITERATION_STARTED, scheduler)
+
+            @default_trainer.on(Events.ITERATION_COMPLETED)
```

Comment:
@sdesrozis you should print on ITERATION_STARTED and before applying lr scheduler

Reply:
It means the comment above in the doc is not relevant ?

Reply:
It is relevant on how to apply the scheduler, but my comment is about when to print the values.